library/webpack-loaders/css-loader.js

```diff
@@ -37,6 +37,7 @@ function createPlugins(loaderOptions, { resolveForImport, resolveForUrlReplace,
         require('postcss-modules-values'),
         !globalScopeBehaviour && require('postcss-modules-local-by-default')(),
         require('postcss-modules-scope')({ generateScopedName: genericNames(isProduction ? '[hash:base64:5]' : '[folder]-[name]-[local]__[hash:base64:5]') }),
+        require('postcss-calc'),
```

Comment:
This does not seem right. It should be at least `require('postcss-calc')()` (extra braces).

Also, do we need this option? https://github.com/postcss/postcss-calc#mediaqueries-default-false

Reply:
It seem postcss itself covers for the mistake and calls the function itself. I will add the parentheses in a next release